src/collective/listingviews/browser/views/templates/layout.pt

```diff
@@ -14,6 +14,9 @@
 
         <div class="listingviews" tal:define="adapter nocall:view/listing_view_adapter;
             batch folderContents | adapter/retrieve_listing_items;">
+
+             <p id="result_count" tal:condition="python:adapter.display_count"><tal:translate i18n:translate="result_count_showing">Showing</tal:translate><tal:translate i18n:translate="result_count"> <span tal:content="adapter/number_of_items" /> </tal:translate><tal:translate i18n:translate="result_count_items">items</tal:translate></p>
```

Comment:
div is better and I think for the purposes of diazo you need to seperate the label from the number with classes so you can customise it

Reply:
are you sure the translation works that way with the dynamic number inside it?